openapi/pacta.yaml

```diff
@@ -944,6 +978,103 @@ components:
         analysis_id:
           type: string
           description: The analysis id to track for the parsing task.
+    HoldingsDate:
+      type: object
+      required:
+        - time
+      properties:
+        time: 
+          type: string 
+          format: date-time
+          description: The time at which the holdings are represented at 
+    IncompleteUpload:
+      type: object
+      properties:
+        ID:
+          type: string # Assuming IncompleteUploadID is a string, otherwise define its structure
+          description: Unique identifier for the incomplete upload
+        Name:
+          type: string
+          description: Name of the upload
+        Description:
+          type: string
+          description: Description of the upload
+        HoldingsDate:
+          $ref: '#/components/schemas/HoldingsDate'
+        CreatedAt:
+          type: string
+          format: date-time
+          description: The time when the upload was created
+        RanAt:
+          type: string
+          format: date-time
+          description: The time when the upload process was run
+        CompletedAt:
+          type: string
+          format: date-time
+          description: The time when the upload was completed
+        FailureCode:
+          type: string
+          description: Code describing the failure, if any
+        FailureMessage:
+          type: string
+          description: Message describing the failure, if any
+        AdminDebugEnabled:
+          type: boolean
```

Comment:
Nit: Why Are All These Properties Inexplicably Capitalized?

Reply:
oops. Because I borrowed the list from the Golang struct.